libnavigation-ui/src/main/java/com/mapbox/navigation/ui/NavigationView.java

```diff
@@ -396,6 +396,7 @@ public void updateCameraRouteOverview() {
    * @param options with containing route / coordinate data
    */
   public void startNavigation(NavigationViewOptions options) {
+    navigationMap.drawRoute(options.directionsRoute());
```

Comment:
I don't think we should have this line here.  `NavigationView` in this API should not draw the route. If the user wants he should be able to call `navigationView.drawRoute(route)`

Reply:
IMO, `NavigationView` should offer a full experience out of the box, this also means drawing a route. Especially to keep compatibility with old codebases. https://github.com/mapbox/mapbox-navigation-android/pull/2563#discussion_r390891178 addresses this as well.

Reply:
I agree.

Reply:
This line can be removed now. This was fixed in https://github.com/mapbox/mapbox-navigation-android/pull/2563.

Reply:
Are we good here? 🤔 @abhishek1508 @LukasPaczos